votepy/k-borda.py

```diff
@@ -1,8 +1,7 @@
-from typing import Iterable, Sequence, List
 from ordinal_election import OrdinalElection, OrdinalBallot
+from typing import Union, List
 
-
-def k_borda(voting: OrdinalElection, size_of_committee: int, number_of_scored_candidates: int) -> OrdinalBallot:
+def k_borda(voting: Union[OrdinalElection, List[int]], size_of_committee: int, number_of_scored_candidates: int) -> OrdinalBallot:
```

Comment:
You can use OrdinalElection | List[int]. I belive it's cleaner. See: https://docs.python.org/3/library/stdtypes.html#types-union

Reply:
Seems good

Reply:
So we need to update the compatible Python versions - `|` syntax is available from the 3.9 Python version.